src/pages/Layout/Layout.tsx

```diff
@@ -117,10 +117,10 @@ const DrawerAppBar = (props: any) => {
       </Box>
       <Box component="main" sx={{ p: 3 }}>
         <Toolbar />
-        <Typography>This is where the pages go!!!!</Typography>
+        <Typography>{children}</Typography>
```

Comment:
Is it necessary to use typography in this one? could it be just children?

Reply:
Good catch!!